### PR TITLE
Fixed Role Check

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#11898

*Description of changes:*
* fixed role check to adhere to api update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/App.spec.ts [hunk 1]
```diff
@@ -140,11 +140,11 @@ const BCOMP_PARTIES = [
     roles: [
       {
         appointmentDate: '2020-07-06',
-        roleType: 'completing party'
+        roleType: 'Completing party'
```
Comment (vysakh-menon-aot) [inline tests/unit/App.spec.ts:143]: p should be Caps

Comment (cameron-freshworks) [reply tests/unit/App.spec.ts:143]: Updated, thanks!

File: src/App.vue [hunk 2]
```diff
@@ -644,17 +644,6 @@ export default {
...
-      // Format party roles from IA
-      application.parties.forEach( // Check each party for roles
-        party => party.roles?.forEach( // Check each role for roleType
-          (role, index) => {
-            if (role.roleType) { // If roleType exists, assign it to parent role index
-              party.roles[index] = role.roleType.toLowerCase()
-            }
-          }
-        )
-      )
```
Comment (severinbeauvais) [inline src/App.vue:656]: No longer needed?

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/App.vue (hunk 2 above)
Comment (cameron-freshworks) [reply src/App.vue:656]: I don't believe so no, but to verify, what state is this triggered. 
Is this after a user completes a Filed IA, that interm state before they have to reload there dashboard?

Comment (cameron-freshworks) [reply src/App.vue:656]: Going to try and verify the structure in the IA. This could still be an edge case

Comment (cameron-freshworks) [reply src/App.vue:656]: Ok, verified 👍

Comment (severinbeauvais) [reply src/App.vue:656]: Did you file an IA to test this?

Comment (severinbeauvais) [reply src/App.vue:656]: > I don't believe so no, but to verify, what state is this triggered. Is this after a user completes a Filed IA, that interm state before they have to reload there dashboard?

Yeah, looks like that scenario.

When the entity is opened using a permanent ID then the IA filing will naturally appear in the ledger.

Comment (cameron-freshworks) [reply src/App.vue:656]: Exactly, you bet. The state where it’s still a temp reg but with a filing in the history.

Comment (severinbeauvais) [reply src/App.vue:656]: You da man.
💯 
😎
